$OpenBSD: patch-src_mod_auth_kerb_c,v 1.1 2013/06/17 19:31:25 ajacoutot Exp $

src/mod_auth_kerb.c:1152: error: 'GSS_KRB5_NT_PRINCIPAL_NAME' undeclared (first use in this function)

--- src/mod_auth_kerb.c.orig	Tue Jun  4 15:38:45 2013
+++ src/mod_auth_kerb.c	Tue Jun  4 15:38:29 2013
@@ -89,6 +89,7 @@
 #include <krb5.h>
 #ifdef HEIMDAL
 #  include <gssapi.h>
+#  include <gssapi/gssapi_krb5.h>
 #else
 #  include <gssapi/gssapi.h>
 #  include <gssapi/gssapi_generic.h>
